akka-stream-tests/src/test/scala/akka/stream/impl/GraphStageLogicSpec.scala

```diff
@@ -32,6 +35,29 @@ class GraphStageLogicSpec extends StreamSpec with GraphInterpreterSpecKit {
     }
   }
 
+  class substreamEmit extends GraphStage[SourceShape[Source[Int, NotUsed]]] {
```

Comment:
Class names should start with a capital letter.

Reply:
Thank you for pointing it out.
I changed class name.